charts/cadc-tap/templates/tap-deployment.yaml

```
@@ -39,18 +67,23 @@ spec:
                 -Dtapschemauser.driverClassName=com.mysql.cj.jdbc.Driver
                 -Dtapschemauser.url=jdbc:mysql://{{ .Values.config.tapSchemaAddress }}/
                 -Dtapschemauser.maxActive=100
-                -Duws.username=postgres
                 -Duws.driverClassName=org.postgresql.Driver
-                -Duws.url=jdbc:postgresql://{{ template "cadc-tap.fullname" . }}-uws-db/
                 -Duws.maxActive=100
+                {{- if .Values.cloudsql.enabled }}
+                -Duws.password=$uwspassword
+                -Duws.username={{ .Values.config.uwsDatabaseUsername }}
+                -Duws.url=jdbc:{{ .Values.config.uwsDatabaseUrl }}
```

If my suggestion above sounded good, this would then be something like:
```suggestion
                -Duws.username={{ .Values.config.cloudsql.database }}
                -Duws.url=jdbc:postgres://localhost:5432/{{ .Values.config.cloudsql.database }}
```

---

I agree and gone with this for now. Perhaps later if we want to open the door for external UWS that is not on cloudsql, we can revisit